### ISSUE TITLE
[DevKit Translator Sample] Azure Function Fails

### ISSUE THREAD

**Issue by darenm** [microsoft/vscode-iot-workbench#738]
Azure IoT Workbench Version 0.10.13
Tried on both Windows 10 and Mac OS.
Created the DevKitTranslator Sample.

After configuring and deploying the Azure Function `DevKitTranslatorFunction` the function throws an exception when trying to access the Azure Speech SDK. Looking at the `Functions.csproj` an old version of the `Microsoft.CognitiveServices.Speech` package is referenced, so I tried updating that, however I get a different, yet similar, issue.

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netcoreapp2.1</TargetFramework>
    <AzureFunctionsVersion>V2</AzureFunctionsVersion>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.Azure.Devices" Version="1.17.1" />
    <PackageReference Include="Microsoft.CognitiveServices.Speech" Version="1.1.0" />
    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="1.0.*" />
  </ItemGroup>
  <ItemGroup>
    <None Update="host.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
    <None Update="local.settings.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      <CopyToPublishDirectory>Never</CopyToPublishDirectory>
    </None>
  </ItemGroup>
</Project>
```

Error for package 1.1.0 - The type initializer for 'Microsoft.CognitiveServices.Speech.Internal.carbon_csharpPINVOKE' threw an exception.

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netcoreapp2.1</TargetFramework>
    <AzureFunctionsVersion>V2</AzureFunctionsVersion>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.Azure.Devices" Version="1.17.1" />
    <PackageReference Include="Microsoft.CognitiveServices.Speech" Version="1.6.0" />
    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="1.0.*" />
  </ItemGroup>
  <ItemGroup>
    <None Update="host.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
    <None Update="local.settings.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      <CopyToPublishDirectory>Never</CopyToPublishDirectory>
    </None>
  </ItemGroup>
</Project>
```

Error for 1.6.0 - unable to load DLL 'Microsoft.CognitiveServices.Speech.core.dll' or one of its dependencies: The specified module could not be found. (Exception from HRESULT: 0x8007007E)

**Comment by zunhu** [comment]
hi, I'm following test doc: https://github.com/Microsoft/vscode-iot-workbench/blob/master/docs/iot-devkit/devkit-translator.md
no issues found in win10.
would you please generate new sample and try it again?
![image](https://user-images.githubusercontent.com/40815873/64402528-e1ec8200-d0a7-11e9-99c7-f3c43433ae6d.png)

**Comment by darenm** [comment]
The image shows you only deployed it, not executed it. The exception occurs when you actually run the function on Azure. I have tried this multiple times.

Thanks,

Daren
________________________________

**Comment by zunhu** [comment]
Hi, thanks for repo this issue, we are in investigating.

**Comment by darenm** [comment]
As in update, I had .net core 3.0 sdk installed. That was being used to compile the assembly locally before deployment. Forcing the use of 2.2 to compile resulted in a function that executed successfully when deployed to Azure.

**Comment by JerryYangKai** [comment]
Hi, we have released workbench 0.10.15 and fixed the bug. You can create a new sample project and check it.
